applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -25,24 +18,23 @@
 import reactor.util.annotation.NonNull;
 
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.EnumSet;
-import java.util.List;
+import java.util.Arrays;
 import java.util.Locale;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
 /**
- * Client that contains all the operations for KeyValues in Azure Configuration Store.
+ * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration
+ * Store.
+ *
+ * @see ConfigurationClientBuilder
+ * @see ConfigurationClientCredentials
  */
 public final class ConfigurationClient extends ServiceClient {
```

Comment:
Can you eagerly rename this class and the builder to `ConfigurationAsyncClient` and `ConfigurationAsyncClientBuilder`?